### DISCUSSION TITLE
Cloud storage integration or Git integration for dsp files

### DISCUSSION THREAD

**Discussion by thopman** [grame-cncm/fausteditorweb#86] [Ideas]
what i would love to see in the faust-ide: cloud integration for saving *.dsp files. I switch computers regularly and it would be super nice to just continue where i left off. Even better would be git integration.

**Comment by dstengle** [comment]
I'd love this too. I've tried making a container to run locally but parts of it don't work as expected (examples don't load). My thought was to somehow load files from the workspace into a local editor.
